cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -149,3 +150,87 @@ def test_new_report_fields(appliance, based_on, request):
     report = appliance.collections.reports.create(**data)
     request.addfinalizer(report.delete_if_exists)
     assert report.exists
+
+
+@pytest.mark.tier(1)
+def test_report_edit_secondary_display_filter(appliance, request, soft_assert):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/6h
+        setup:
+            1. Create/Copy a report with secondary (display) filter.
+        testSteps:
+            1. Edit the secondary filter and test if the report was updated.
+        expectedResults:
+            1. Secondary filter must be editable and it must be updated.
+
+    Bugzilla:
+        1565171
+    """
+    report_data = {
+        "title": "Testing report",
+        "menu_name": "testing report",
+        "base_report_on": "Instances",
+        "report_fields": [
+            "Active",
+            "EVM Custom Attributes : Name",
+            "EVM Custom Attributes : Region Description",
+            "EVM Custom Attributes : Region Number",
+            "Name",
+        ],
+        "consolidation": {
+            "group_records": [
+                "EVM Custom Attributes : Name",
+                "EVM Custom Attributes : Region Description",
+                "EVM Custom Attributes : Region Number",
+            ]
+        },
+        "filter": {
+            "primary_filter": "fill_field(Instance : Active, IS NOT NULL)",
+            "secondary_filter": "fill_field(EVM Custom Attributes : Name, INCLUDES, A)",
+        },
+    }
+
+    report = appliance.collections.reports.create(**report_data)
```

Comment:
Since the initial report creation is part of setup, I would expect it to be a fixture. What do you think?

Reply:
I think it should be a fixture as well. We had something similar in v2v, which we moved to fixtures. That helped us avoiding huge `data` dictionaries in test cases as we have above here.